Documentation/ceph-object.md

```diff
@@ -105,6 +105,43 @@ To directly retrieve the secrets:
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep AccessKey | awk '{print $2}' | base64 --decode
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep SecretKey | awk '{print $2}' | base64 --decode
 ```
+## Create an Object Bucket Claim
+
+Using Object Bucket Claim(otherwise known as obc), a bucket will be provisioned from object store for the application. It can consume already existing buckets or else new bucket will be created. For this first a storage class need to be defined and then use it to create object bucket claim, all using prefered settings. See the [Object Bucket Claim CRD](ceph-object-bucket-claim-crd.md) for more detail on the settings available for a `CephObjectBucketClaim`.
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: rook-ceph-bucket
+provisioner: ceph.rook.io/bucket
+reclaimPolicy: Delete
+parameters:
+  objectStoreName: my-store
+  objectStoreNamespace: rook-ceph
+  region: us-east-1
+```
+```bash
+# Create the Storage class for bucket provisioner
+kubectl create -f storageclass-bucket.yaml
+```
+```yaml
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: ceph-bucket
+spec:
+  bucketName: ceph-bkt
+  storageClassName: rook-ceph-bucket
+```
+
+After creating the OBC, a secret and config map will be configure to access the object store with similar name. The creditionals can be fetched using following commands :
```

Comment:
Change to:

"After creating the OBC, a secret and configmap are created with the same name as the OBC and in the same namespace. The secret contains credentials used by the application pod to access the bucket. The configmap contains bucket endpoint information and is also consumed by the pod.

The following commands extract key pieces of information from the secret and configmap:"

Reply:
Done